README.md

```diff
@@ -1,2 +1,16 @@
 # comp110-worksheet-B
```

Comment:
Flowchart looks mostly OK. Quite a few typos in the text, but the logic is easy enough to follow.